app/brave_main_delegate.cc

```diff
@@ -93,3 +99,16 @@ void BraveMainDelegate::PreSandboxStartup() {
     brave::InitializeResourceBundle();
   }
 }
+
+bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
+  base::CommandLine& command_line =
+      *base::CommandLine::ForCurrentProcess();
+  command_line.AppendSwitch(switches::kEnableTabAudioMuting);
+
+  std::stringstream enabled_features;
+  enabled_features << features::kEnableEmojiContextMenu.name
+    << "," << features::kDesktopPWAWindowing.name;
+  command_line.AppendSwitchASCII(switches::kEnableFeatures,
```

Comment:
will this work correctly with other command line features or will it do something like `existingswitchEnableEmojieContextMenu,DesktopPWAWindowing`?

Reply:
if it does work correctly, maybe it would be better to call `AppendSwitchASCII` once for each switch?

Reply:
> will this work correctly with other command line features or will it do something like existingswitchEnableEmojieContextMenu,DesktopPWAWindowing?

- I tried enabling other things in about:flags and it doesn't conflict
- I also tried disabling things enabled here explicitly and that worked too.
- I also tried starting with --enable_features=some-other-feature and that worked too

> if it does work correctly, maybe it would be better to call AppendSwitchASCII once for each switch?

If it did work, that would be different, but not better for any reason.
It doesn't work though if you append twice.  Yet as noted above it allows you to still use the command line for some reason passed when you start and also append the switch here, but not append 2 switches for some reason.

Reply:
That's odd, you would think that it should just keep appending if it works with an existing `--enable_features` value. That just seemed a cleaner than building the string and then appending it, but as long as it works with an existing `--enable_features` value then it's ok